src/less/variables/variables.less

```diff
@@ -19,51 +19,14 @@
 @_page-grid-outside-margins-small: 16px;
 @_page-grid-outside-margins-large: 32px;
 @_page-grid-outside-margins-max: 48px;
-@_page-grid-column-gaps-small: @spacing-100;
-@_page-grid-column-gaps-large: @spacing-200;
-@_page-grid-row-gaps-small: @spacing-100;
-@_page-grid-row-gaps-large: @spacing-200;
+@_page-grid-column-gaps-small: var(--spacing-100);
+@_page-grid-column-gaps-large: var(--spacing-200);
+@_page-grid-row-gaps-small: var(--spacing-100);
+@_page-grid-row-gaps-large: var(--spacing-200);
 
 @marketsans-url: "https://ir.ebaystatic.com/cr/v/c1/market-sans/v1.0/";
 @marketsans-fontname: "Market Sans";
 @marketsans-filename-regular: "MarketSans-Regular-WebS";
 @marketsans-filename-bold: "MarketSans-SemiBold-WebS";
 
 @font-family-default: @marketsans-fontname, Arial, sans-serif;
-
-@font-weight-regular: 500;
```

Comment:
We can't remove these until next major version.

Reply:
Done. rolled back.